prime-router/src/main/kotlin/tokens/Jwk.kt

```diff
@@ -33,26 +34,46 @@ import java.security.interfaces.RSAPublicKey
  */
 @JsonInclude(JsonInclude.Include.NON_NULL)
 class Jwk(
-    val kty: String, // Key type (Alg family).  eg, RSA, EC, oct
-    val use: String? = null, // Intended use.  eg, sig, enc.
+    @field:Schema(description = "The family of cryptographic algorithms used with the key.  eg, RSA, EC, oct")
```

Comment:
This is just getting the generated doc to match the existing api file.  Looking at this more closely though it would likely make sense to create something like `JwkDto` in the API and just have the specific fields.